pmpsui/ui/trans_override.ui

```diff
@@ -806,116 +537,656 @@
+          <string>Usage Notes</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="Line" name="line_2">
+         <property name="orientation">
+          <enum>Qt::Horizontal</enum>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="label">
+         <property name="text">
+          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This override makes the beam &lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;less safe&lt;/span&gt; by &lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;increasing&lt;/span&gt; the gas attenuator &lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;transmission&lt;/span&gt; and allowable &lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;beam class&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;The PMPS transmission parameters were set with an assumed maximum credible energy of 5 mJ. When this assumption is wildly wrong, you may use this override to set a new maximum credible energy and globally adjust the requirements for a shift duration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+         </property>
+         <property name="wordWrap">
+          <bool>true</bool>
+         </property>
+         <property name="indent">
+          <number>10</number>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <spacer name="horizontalSpacer_6">
+         <property name="orientation">
+          <enum>Qt::Horizontal</enum>
+         </property>
+         <property name="sizeType">
+          <enum>QSizePolicy::Fixed</enum>
+         </property>
+         <property name="sizeHint" stdset="0">
+          <size>
+           <width>40</width>
+           <height>20</height>
+          </size>
+         </property>
+        </spacer>
+       </item>
+       <item>
+        <widget class="QLabel" name="label_32">
+         <property name="font">
+          <font>
+           <pointsize>12</pointsize>
+           <weight>75</weight>
+           <bold>true</bold>
+          </font>
+         </property>
+         <property name="toolTip">
+          <string/>
+         </property>
+         <property name="text">
+          <string>Math</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="Line" name="line_3">
+         <property name="orientation">
+          <enum>Qt::Horizontal</enum>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="label_2">
+         <property name="text">
+          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Transmission adjustments are proportional. A maximum credible energy of 0.5 mJ would result in all transmissions being multipled by 10.&lt;/p&gt;&lt;p&gt;The formula is: t_new = t_old * 5mJ / e_max&lt;/p&gt;&lt;p&gt;t_new is capped at 1.0 = 100% transmission.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Have you tried using html formatting to actually get subscripts?  I think actual LaTeX is probably a step too far but sub/superscripting variable names when applicable might help readability